lmdeploy/model.py

```diff
@@ -18,6 +18,31 @@ class SamplingParam:
     repetition_penalty: float = 1.0
 
 
+@dataclasses.dataclass
+class ModelConfig:
+    """parameters for chat templates."""
+    model_name: str  # determine which chat template will be applied
+    system: Optional[str] = None  # begin of the system prompt
```

Comment:
Shouldn't the default values of the attributes of string type be empty strings?

Reply:
The default value should be our settings in model.py. `None` is a flag here to determine whether we should modify default settings.

Reply:
Suppose the user wants to disable our default settings. He will set `meta_instructions=''`.